Zend/zend_attributes.stub.php

```diff
@@ -2,18 +2,62 @@
 
 /** @generate-class-entries */
 
+#[Attribute(Attribute::TARGET_CLASS)]
 final class Attribute
 {
+    /**
+     * @var int
+     * @cname ZEND_ATTRIBUTE_TARGET_CLASS
+     */
+    const TARGET_CLASS = UNKNOWN;
```

Comment:
can you please add the `public` modifier as well? Or is it just my preference to use them? 🤔

Reply:
Probably your preference :-D In fact, I don't really see why ext code should ever publish non-public consts.